2

```diff
@@ -0,0 +1,13 @@
+Update Snapshot and upload CRD status in doc.
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# On branch lipingx-doc-update-July-24
+# Changes to be committed:
+#	modified:   docs/vanilla.md
+#
+# Untracked files:
+#	.DS_Store
+#	docs/.DS_Store
+#	set-unit-env.sh
+#
```

Comment:
No sure why this showed up.  You may want to rebase your branch.

Reply:
Removed this file